.gitignore

```diff
@@ -18,7 +18,6 @@ npm-debug.log*
 .sass-cache/
 .tmp/
 .versions/
-dist/
```

Comment:
hello, why do you want to have dist folder included in git?

Reply:
Hi, i included the dist folder only to import easily the modified library from my github repo to my project. I opened the pull request only for the commit [https://github.com/lucaboy/capacitor-secure-storage-plugin/commit/a742bb2f17cec1f21ef5a9af2e2e9f7121d5d4f8](a742bb2f17cec1f21ef5a9af2e2e9f7121d5d4f8).